AudioSystem/AudioConfig.cs

```diff
@@ -6,9 +6,16 @@
 
 namespace DUCK.AudioSystem
 {
+	public abstract class AbstractAudioConfig : ScriptableObject
+	{
+		public abstract AudioConfig[] AudioConfigs { get; }
+	}
+
 	[CreateAssetMenu(menuName = "Audio Config (Fancy™)", order = 220)] // Right after Audio Mixer
```

Comment:
I don't even need to look at the `git log` to know who the original author of this particular line was @Rollo-Liu

Reply:
`Signature`